addons/account_edi/models/account_move.py

```diff
@@ -618,7 +618,7 @@ def _get_update_invoice_from_attachment_decoders(self, invoice):
 
     def action_process_edi_web_services(self):
```

Comment:
Add param with_commit = True
remove _action_process_edi_web_services from l10n_pe_edi

Reply:
one is in a test, the other not

Reply:
Yes, @smetl suggested that once the with_commit parameter was available, the `_action_process_edi_web_services` would no longer be necessary.